bakery/src/scripts/link_single.py

```diff
@@ -35,6 +35,21 @@ def create_canonical_map(input_dir):
     return canonical_map
 
 
+def create_slug_map(input_dir):
+    """Create a slug:uuid map from baked metadata files"""
+    baked_metadata_files = Path(input_dir).glob("*.baked-metadata.json")
```

Comment:
Seems like we are walking the file system twice (see line with `for root, _, files in os.walk(baked_meta_dir):`) to get the same metadata files.

Reply:
Aha, you fixed it in the next commit. Disregard.